### PR TITLE
add scenario 001

### PR DESCRIPTION
**Pull request by sintinilorenzo**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

**Review by chengoconnell — APPROVED** [review]
nice!

File: cypress/plugins/createRepo.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      PrismicSharedConfigManager.get().cookies
+    )
+    .catch(() => {});
+
+  await client.createRepository(DOMAIN_NAME, Frameworks.next);
+
+  return;
+};
+
+deleteAndCreate();
```
Comment (MarcMcIntosh) [inline cypress/plugins/createRepo.ts:37]: how does this get called by cypress?
Would the `on` method be useful here?
https://docs.cypress.io/api/plugins/writing-a-plugin#on

File: cypress/integration/user-flows/scenario_001.spec.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
+import path from "path";
+
+describe("I am a new SM user (with Next) who wants to create a Custom Type with fields, and then save and push it to Prismic.", () => {
```
Comment (MarcMcIntosh) [inline cypress/integration/user-flows/scenario_001.spec.js:3]: 👏

File: cypress/plugins/createRepo.ts (hunk 1 above)
Comment (sintinilorenzo) [reply cypress/plugins/createRepo.ts:37]: it's not called by cypress, it's called in the cypress setup .sh script

Comment (MarcMcIntosh) [reply cypress/plugins/createRepo.ts:37]: cool, maybe add that as a comment in the file :)

Comment (sintinilorenzo) [reply cypress/plugins/createRepo.ts:37]: done :)

Comment (MarcMcIntosh) [reply cypress/plugins/createRepo.ts:37]: nice :)

**Review by MarcMcIntosh — APPROVED** [review]

**Pull request merged**